src/containers/flows/Raffle/Drop.constants.ts

```diff
@@ -3,16 +3,16 @@ const currentTime = new Date().getTime();
 // Temporary values
 export const RAFFLE_START_TIME = currentTime - 10000;
 export const RAFFLE_END_TIME = currentTime - 50000;
-// export const SALE_START_TIME = currentTime + 10000;
-// export const PRIVATE_SALE_END_TIME = currentTime + 90000;
+export const SALE_START_TIME = currentTime + 10000;
+export const PRIVATE_SALE_END_TIME = currentTime + 90000;
 
 // const PUBLIC_SALE_START_TIME = currentTime + 100000;
 // const SALE_START_BLOCK = 13719840;
 
 // Hardcoded event times
 // export const RAFFLE_START_TIME = 1650589200000;
 // export const RAFFLE_END_TIME = 1650762000000;
-export const SALE_START_TIME = 1671652800000;
-export const PRIVATE_SALE_END_TIME = 1671739200000;
+// export const SALE_START_TIME = 1671652800000;
+// export const PRIVATE_SALE_END_TIME = 1671739200000;
 // export const PUBLIC_SALE_START_TIME = 1651280400000;
 //
```

Comment:
nit: remove comments

Reply:
kept them for switching between test and real values